rosetta-source/src/main/rosetta/base-staticdata-asset-common-func.rosetta

```diff
@@ -0,0 +1,19 @@
+namespace cdm.base.staticdata.asset.common
+version "${project.version}"
+
+func OrdinalExists: <"Requires that, if multiple classification attributes are present, they contain an  ordinal so that they can be sorted. The condition checks that the number of classfication attributes is the same as the number of ordinals. Since each classification attributes can only contain one ordinal attribute, the logic effectively enforces that whenever more than one classification attribute exists, all classification attributes will have an ordinal.">
+    inputs:
+        productTaxonomy ProductTaxonomy (0..*)
+    output:
+        result boolean (0..1)
+    set result:
+        productTaxonomy
+            filter source = TaxonomySourceEnum -> ISDA or source = TaxonomySourceEnum -> EMIR
```

Comment:
Why do you need this filter? It looks ad-hoc...
Is that because in other cases, `ordinal` does not exist and so the condition fails? If that is the case, you should instead test whether 'ordinal' exists. See below.

Reply:
I think the idea here was more to restrict applicability to the classification systems we know that work this way. I agree it may be too ad-hoc... We can remove the filter, which will imply that we assume that in all classification systems never mix a hierarchical classification (with `ordinal`) and a non-hierarchical one (without `ordinal`). I think it could be a reasonable assumption but we added the filter for safety.